tests/chains/ethereum/scripts/cmd/balance/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/big"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/datachainlab/yui-relayer-build/tests/chains/ethereum/scripts/cmd/helper"
+	"github.com/spf13/cobra"
+)
+
+const (
+	relayer = 0
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "wallet",
+	Short: "wallet command",
+	Long:  "wallet command walletIndex",
+	Run: func(cmd *cobra.Command, args []string) {
+		configDir := args[0]
+		walletIndex, err := strconv.ParseInt(args[1], 10, 64)
+		if err != nil {
+			log.Println(err)
+			os.Exit(1)
+		}
+		simpleTokenAddress := args[2]
+		ics20TransferBankAddress := args[3]
+		ics20BankAddress := args[4]
+		balanceA, balanceB := balanceOf(configDir, walletIndex, simpleTokenAddress, ics20TransferBankAddress, ics20BankAddress)
+		fmt.Printf("%d,%d\n", balanceA, balanceB)
+	},
+}
+
+func main() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func balanceOf(configDir string, index int64, simpleTokenAddress, ics20TransferBankAddress, ics20BankAddress string) (*big.Int, *big.Int) {
+	chainA, chainB, err := helper.InitializeChains(configDir, simpleTokenAddress, ics20TransferBankAddress, ics20BankAddress)
+	if err != nil {
+		log.Println("InitializeChains Error: ", err)
+		os.Exit(1)
```

Comment:
You can use `log.Fatalln` (https://pkg.go.dev/log#Fatalln) instead of "print and exit".
Another option is that balanceOf returns err and callers handle it.

Reply:
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/03393f65363e07b4b2c7ee5b2962f1b7d2fcd2f1
Changed to use `log.Fatalln`